checkov/common/checks_infra/solvers/attribute_solvers/length_equals_attribute_solver.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Optional, Any, Dict
+from collections.abc import Sized
+from checkov.common.checks_infra.solvers.attribute_solvers.base_attribute_solver import BaseAttributeSolver
+from checkov.common.graph.checks_infra.enums import Operators
+from checkov.common.util.type_forcers import force_integer
+
+
+class LengthEqualsAttributeSolver(BaseAttributeSolver):
+    operator = Operators.LENGTH_EQUALS
+
+    def __init__(self, resource_types: List[str], attribute: Optional[str], value: Any) -> None:
+        super().__init__(resource_types=resource_types, attribute=attribute, value=value)
+
+    def _get_operation(self, vertex: Dict[str, Any], attribute: Optional[str]) -> bool:
+        if vertex.get(attribute) is None:  # type:ignore[arg-type]  # due to attribute can be None
+            return False
+
+        attr = vertex.get(attribute)  # type:ignore[arg-type]  # due to attribute can be None
+        # if this value contains an underendered variable, then we cannot evaluate the check,
+        # so return True (since we cannot return UNKNOWN)
+        if self._is_variable_dependant(attr, vertex['source_']):
+            return True
+
+        if isinstance(attr, Sized):
+            return len(attr) == force_integer(self.value)
```

Comment:
```suggestion
            return len(attr) == force_int(self.value)
```
don't need to add a new function, it already exists in the same file 😄

Reply:
missed that ^^